src/Builder/ListBuilder.php

```diff
@@ -50,6 +50,12 @@ public function buildField(?string $type, FieldDescriptionInterface $fieldDescri
     {
         if (null === $type) {
             $guessType = $this->guesser->guess($fieldDescription);
+            if (null === $guessType) {
```

Comment:
If every time we are returning null we throw an exception, wouldnt it be better to only return non null values or throw exception inside the guesser?

Reply:
I made https://github.com/sonata-project/SonataAdminBundle/pull/6987 because the TypeGuesserChain could return null...

If you have a TypeGuesserChain with no guesser how can you return a type ?
And the Guesser interface from Symfony was allowing `null` as return type...

The Guesser from this bundle never return null, but just in case, I added the exception...